app/assets/stylesheets/activities.scss

```diff
@@ -0,0 +1,9 @@
+.activities_history {
+	border: 1px solid dimgrey;
```

Comment:
Plz use hexcode instead of name.
Also, plz amke this a variable. If you think this color will be used only in this file, make it a variable in this file.
Else, create a file called theme.scss and add all variables in one file.
Plz have a look at `CodeAstra/codeacademy` repo.